src/scripts/generate-hasura-schema.ts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
```

Comment:
This is important to have a descriptive ReadMe file. so for everyone is clear how it works and what it will generate